.coderabbit.yaml

```
@@ -0,0 +1,141 @@
+        - Documentation value for design system users
+    
+    # Design Tokens
+    - path: "**/packages/tokens/src/**/*.ts"
+      instructions: |
+        Review design token changes for:
+        - Backward compatibility with existing components
+        - Consistent naming conventions following established patterns
+        - Proper TypeScript exports and type definitions
+        - Impact on overall design system consistency
+        - Documentation of token usage and purpose
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)